SQL/tests/syntax/syntax_test_postgres.psql

```diff
@@ -551,3 +561,90 @@ value := E'\u1234 \U9876A001'
 --         ^^^^^^ constant.character.escape
 --               ^ - constant
 --                ^^^^^^^^^^ constant.character.escape
+
+CREATE OR REPLACE FUNCTION public."AutAppB_getBatchOrders"(
+    "@sBatchTy" character varying,
+--^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.statement.create.sql meta.function.parameters.sql meta.group.sql
+--  ^^^^^^^^^^^ variable.parameter.sql
+--  ^ punctuation.definition.identifier.begin.sql
+--            ^ punctuation.definition.identifier.end.sql
+--              ^^^^^^^^^^^^^^^^^ storage.type.sql
+--                               ^ punctuation.separator.sequence.sql
+    "@lBatchNum" character varying,
+    "@ValidAccess" character varying DEFAULT NULL::character varying)
+--                                   ^^^^^^^ storage.modifier.sql
+--                                           ^^^^ constant.language.null.sql
+--                                               ^^ keyword.operator.cast.psql
+--                                                 ^^^^^^^^^^^^^^^^^ storage.type.sql
+--                                                                  ^ punctuation.section.group.end.sql
+    RETURNS TABLE("sCliTS" character varying, "sCustID" character varying, "sCliAcctNum" character varying, "cCrdOrdAmt" numeric, "bLnRqst" smallint, "sDcsnCd" character varying, "sCrdTy" character varying, "cDecision" numeric, "sRsnCd1" character varying, "sPONum" character varying, "sApprNum" character varying, "dtShip" timestamp without time zone, "dtExp" timestamp without time zone, "sCliCmmt" character varying, "sCustNm" character varying, "sDesc" character varying, "sDcsnDesc" character varying, "sCrdDesc" character varying, "sRsnDesc" character varying, "sCliTrmCd" character varying, "lCARNum" integer, "sNotes" character varying, "sStsCd" character varying) 
+    LANGUAGE 'plpgsql'
+    COST 100
+    VOLATILE PARALLEL UNSAFE
+    ROWS 1000
+
+AS $BODY$
+
+# variable_conflict use_column
+DECLARE
+    "@@iIndex" INT;
+    "@@iTotal" INT;
+    "@@bLog" SMALLINT;
+    "@@dtLate" TIMESTAMP;
+    "@@dtEff" TIMESTAMP;
+BEGIN
+  IF "isAuth"("@ValidAccess") = 1 THEN
+    "@sBatchTy" = left("@sBatchTy", 1);
+    "@lBatchNum" = left("@lBatchNum", 10);
+
+    CREATE TEMPORARY TABLE "#ProcParms" (
+--  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.statement.create.sql - comment
+--  ^^^^^^ keyword.other.ddl.sql
+--         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.table.sql
+--         ^^^^^^^^^ keyword.other.ddl.sql
+--                   ^^^^^ keyword.other.ddl.sql
+--                         ^^^^^^^^^^^^ entity.name.struct.table.sql
+--                         ^ punctuation.definition.identifier.begin.sql
+--                                    ^ punctuation.definition.identifier.end.sql
+--                                      ^ meta.group.table-columns.sql punctuation.section.group.begin.sql
+         "MySPID" VARCHAR(15)
+        ,"lBatchNum" INT
+        ,"bLog" SMALLINT
+        ,"bCPU" SMALLINT
+        ,"dtEff" TIMESTAMP WITHOUT TIME ZONE
```

Comment:
Note all quoted identifiers being table names. They are scoped literal string due to `expressions` being included into column declaration list without any restrictions.